### PR TITLE
PMM-6166 Switch CI to Ubuntu

### PR DESCRIPTION
**Pull request by AlekSi**

### CODE REVIEW

**Comment by it-percona** [comment]
[CLA assistant check](https://cla-assistant.percona.com:443/percona-platform/dbaas-controller?pullRequest=5) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.percona.com:443/percona-platform/dbaas-controller?pullRequest=5) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.percona.com:443/check/percona-platform/dbaas-controller?pullRequest=5) it.</sub>

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,10 +16,12 @@ jobs:
...
     strategy:
       matrix:
-        # FIXME add ubuntu-latest, remove macos-latest: https://jira.percona.com/browse/PMM-6166
-        os: [macos-latest]
+        os: [ubuntu-latest]
         go-version: [1.14.x]
-        kubernetes_version: [1.16.8]  # TODO add other versions
+        kubernetes_version:
+          # https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html
+          - 1.15.11
```
Comment (BupycHuk) [inline .github/workflows/ci.yml:23]: Why did you add older version?

Comment (AlekSi) [reply .github/workflows/ci.yml:23]: That's what our operator supports on EKS: https://www.percona.com/doc/kubernetes-operator-for-pxc/System-Requirements.html
